www/js/metrics-factory.js

```diff
@@ -192,7 +192,7 @@ angular.module('emission.main.metrics.factory',
       mode = 'WALKING';
     }
     let currentMETs = cc.getMETs();
-    if (!currentMETs[mode]) {
+    if (!currentMETs?.[mode]) {
```

Comment:
I don't think that this is the correct fix. Instead, we should just disable the dashboard screen completely unless we are in the MULTILABEL mode.

Please see https://github.com/e-mission/e-mission-phone/pull/938, https://github.com/e-mission/e-mission-phone/pull/940 and https://github.com/e-mission/e-mission-phone/pull/941 for the final fixes

Reply:
reverted 924fdd38d142aa52150366a507dddab4123b7702